sagemaker-fundamentals/create-endpoint/container/src/serve

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
+
+# This file implements the scoring service shell. You don't necessarily need to modify it for various
```

Comment:
You don't necessarily need to modify it for various algorithms. --> You can modify it as need be for your algorithms.

Reply:
done